src/admin/components/SearchResult/index.js

```diff
@@ -4,8 +4,9 @@ import { Link } from 'react-router';
 import { sprintf } from 'jed';
 
 import { ngettext } from 'core/utils';
+import SearchResult from 'amo/components/Search/SearchResult';
 
-import './style.scss';
+import 'amo/components/Search/SearchResult/style.scss';
```

Comment:
I'm fine with that. I am doing what was done in the Admin app but that makes more sense to me too.

I would also rename `Search/SearchResult/index.js` to `Search/SearchResult.js`, in that case. But I like that more myself.